src/query/storage/promremote/query_converter_test.go

```diff
@@ -34,67 +34,122 @@ import (
 	xtime "github.com/m3db/m3/src/x/time"
 )
 
-func TestQueryConverter(t *testing.T) {
+func TestWriteQueryConverter(t *testing.T) {
 	now := xtime.Now()
-	tcs := []struct {
-		name     string
-		input    storage.WriteQueryOptions
-		expected *prompb.WriteRequest
-	}{
-		{
-			name: "converts",
-			input: storage.WriteQueryOptions{
+	dp := ts.Datapoint{
+		Timestamp: now,
+		Value:     42,
+	}
+	tag := models.Tag{
+		Name:  []byte("test_tag_name"),
+		Value: []byte("test_tag_value"),
+	}
+	convertedToLabel := prompb.Label{
+		Name:  "test_tag_name",
+		Value: "test_tag_value",
+	}
+	covertedToSample := prompb.Sample{
+		Timestamp: now.ToNormalizedTime(time.Millisecond),
+		Value:     42,
+	}
+
+	t.Run("single datapoint", func(t *testing.T) {
```

Comment:
I think you should resurrect the test table if you have multiple test cases now.